src/jit/compiler.h

```diff
@@ -1176,6 +1176,10 @@ struct FuncInfoDsc
     BYTE                unwindCodes[offsetof(UNWIND_INFO, UnwindCode) + (0xFF*sizeof(UNWIND_CODE))];
     unsigned            unwindCodeSlot;
 
+#define MAX_CFICODE_INDEX   0x200
+    BYTE                cfiCodes[(MAX_CFICODE_INDEX * sizeof(CFI_CODE))];
+    unsigned            cfiCodeIndex;
+
```

Comment:
I'm reluctant to introduce #ifdef although strictly it is not used for Windows. In fact, we could encode SEH CFI similar to this logic instead of creating UNWIND_BLOB ourselves.
MAX_CFICODE_INDEX is also used to assert we don't overflow when we create an CFI.